docs/language.rst

```diff
@@ -177,6 +177,128 @@ Lists of primitive types are also primitive types: ``string[]``, ``number[]``, `
     # incorrect, can't assign to dict after construction
     # a["otherkey"] = "othervalue"
 
+Strings
++++++++
+
+
+There are four kinds of strings in the Inmanta language:
+
+- regular strings
+
+.. code-block:: inmanta
+
+    regular_string_1 = "This is...\n...a basic string."
+
+    # Output when displayed:
+    # This is...
+    # ...a basic string.
+
+
+    regular_string_2 = 'This one too.'
+
+    # Output when displayed:
+    # This one too.
+
+- multi-line strings
+
+It is possible to make a string span multiple lines by triple quoting it e.g.:
+
+.. code-block:: inmanta
+
+    multi_line_string = """This
+    string
+    spans
+    multiple
+    lines"""
+
+    # Output when displayed:
+    # This
+    # string
+    # spans
+    # multiple
+    # lines
+
+.. note::
+    Unlike python's multi-line strings, only double quotes are supported to define a multi-line string i.e. ``"""`` is
+    valid, but ``'''`` is not.
```

Comment:
I was surprised by this

Reply:
Yes. Me too. I created [a ticket](https://github.com/inmanta/inmanta-core/issues/6488) to support this. In the meantime we just document well that single quotes are not supported.